extras/installer.sh

```diff
@@ -277,6 +277,9 @@ fi
 
 echo
 read -e -p "Directory to install into: " -i "/opt/plexupdate" FULL_PATH
+if [[ "$FULL_PATH" == *"~"* ]]; then # If path contains a tilde
```

Comment:
I didn't even think about ~user as an option. I think it's still broken because if I put in `~jon/plex` I get `/home/jonjon/plex`